features/org.eclipse.pde-feature/feature.xml

```diff
@@ -28,6 +28,13 @@
       <import plugin="org.bndtools.templates.template"/>
       <import plugin="biz.aQute.repository"/>
       <import plugin="bndtools.jareditor"/>
+      <import plugin="org.osgi.test.common"/>
```

Comment:
Should probably become a dedicated feature for easier inclusion in other target platforms and to have an opt-in.